### PR TITLE
Improve consistency of and correct errors in Dutch translation

### PR DESCRIPTION
**Pull request by Flitskikker**

### CODE REVIEW

File: src/Languages/nl-NL.xml [hunk 1]
```diff
@@ -2269,18 +2270,18 @@ Probeer het later opnieuw a.u.b.</XDownloadFailed>
...
     <Title>Lange teksten opdelen</Title>
     <SingleLineMaximumLength>Maximale regellengte</SingleLineMaximumLength>
-    <LineMaximumLength>Maximale tekstlengte</LineMaximumLength>
-    <LineContinuationBeginEndStrings>Regelvoortzetting begin/einde tekst</LineContinuationBeginEndStrings>
+    <LineMaximumLength>Maximale titellengte</LineMaximumLength>
+    <LineContinuationBeginEndStrings>Tekens voor in-/uitpunten titels</LineContinuationBeginEndStrings>
     <NumberOfSplits>Afbrekingen: {0}</NumberOfSplits>
     <LongestSingleLineIsXAtY>Langste regellengte is {0} in tekst {1}</LongestSingleLineIsXAtY>
-    <LongestLineIsXAtY>Langste tekstlengte is {0} in tekst {1}</LongestLineIsXAtY>
+    <LongestLineIsXAtY>Langste titellengte is {0} in tekst {1}</LongestLineIsXAtY>
```
Comment (xylographe) [inline src/Languages/nl-NL.xml:2277]: **`in tekst`** ⇒ **`in titel`** ?

File: src/Languages/nl-NL.xml [hunk 2]
```diff
@@ -1694,29 +1695,29 @@ hetzelfde ondertitelbestand bewerken (samenwerking)</Information>
...
+    <WordBoundary>Woordgrens (\b)</WordBoundary>
+    <NonWordBoundary>Niet-woordgrens (\B)</NonWordBoundary>
     <NewLine>Regeleinde (\r\n)</NewLine>
     <NewLineShort>Regeleinde (\n)</NewLineShort>
     <AnyDigit>Elk cĳfer (\d)</AnyDigit>
     <NonDigit>Geen cĳfer (\D)</NonDigit>
     <AnyCharacter>Elk teken (.)</AnyCharacter>
     <AnyWhitespace>Elke witruimte (\s)</AnyWhitespace>
-    <NonSpaceCharacter>Geen witruimte (\S)</NonSpaceCharacter>
+    <NonSpaceCharacter>Witruimte zonder breedte (\S)</NonSpaceCharacter>
```
Comment (xylographe) [inline src/Languages/nl-NL.xml:1720]: [Regular Expression Language – Character Classes](https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference#character-classes):
```
\s    Matches any white-space character.
\S    Matches any non-white-space character.
```

File: src/Languages/nl-NL.xml (hunk 1 above)
Comment (Flitskikker) [reply src/Languages/nl-NL.xml:2277]: In a lot of strings, "line" refers to the whole subtitle. I chose to use "titel" there for clarification.
For example: "Selected lines" was translated as "Tekstselectie", which implies to refer to the selected text, while the source was referring to the currently selected subtitles. In those cases, I translated it as "Geselecteerde titels".

File: src/Languages/nl-NL.xml (hunk 2 above)
Comment (Flitskikker) [reply src/Languages/nl-NL.xml]: Good catch! Reverted.

**Pull request merged**